src/deploy-events.js

```diff
@@ -1,15 +1,19 @@
-const fs = require('node:fs');
-const path = require('node:path');
-
 module.exports = (client) => {
-  const eventsPath = path.join(__dirname, 'events');
-  const eventFiles = fs
-    .readdirSync(eventsPath)
-    .filter((file) => file.endsWith('.js'));
+  const guildScheduledEventCreate = require('./events/guildScheduledEventCreate');
+  const interactionCreateEvent = require('./events/interactionCreate');
+  const pollVotationResultEvent = require('./events/pollVotationResults');
+  const qaMetionEvent = require('./events/qaMention');
+  const readyEvent = require('./events/ready');
 
-  for (const file of eventFiles) {
-    const event = require(path.join(eventsPath, file));
+  const eventList = [
+    guildScheduledEventCreate,
+    interactionCreateEvent,
+    pollVotationResultEvent,
+    qaMetionEvent,
+    readyEvent,
+  ];
```

Comment:
The imports and the eventList could be outside of the function, since those are static: only changes when you changes the code, but are not modified by any function, and they determine what is the behaviour of the program